bundles/core/org.eclipse.smarthome.core.thing/OSGI-INF/SystemChannelTypeProvider.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
the file should be DefaultSystemChannelTypeProvider.xml as it registers the OSGi service for the DefaultSystemChannelTypeProvider.